### DISCUSSION TITLE
ferdium for windows 7?

### DISCUSSION THREAD

**Discussion by horia-m** [ferdium/ferdium-app#1669] [Q&A]
Hi, I would like to install ferdium on windows 7 x64, which version is working? and how can I download it from the repository? 
thank you! 
(the actual version 6.7.2 does not work, I tried it)

**Comment by SpecialAro** [comment]
Hello @horia-m 

We dropped the support for Windows 7 several versions ago (can't quite remember which - you can check which Electron version supports Windows 7 and you can then know which version of Ferdium works by searching for the corresponding tag).

I really advise against using an older version of Ferdium because it has several vulnerabilities that have been fixed (one of the biggest ones is the webp CVE that haunted several applications).

Also, Windows 7 reached its end of life in 2015 (almost 10 years ago!) and its extended support stopped in 2020 (4 years ago!) - so you should not be using Windows 7 if you can upgrade.

**Comment by steebs** [comment]
I too am a windows 7 user and I will NEVER make the change to WinDOZE 10 or later.   All later versions of Windoze are inferior to Windows 7.   For a start, how many operating systems allow developers to tinker with your computer WITHOUT YOUR PERMISSION!

For you to virtually demand that users move away from Windows 7 shoudl be illegal.   It is certainly unethical for you to make these demands.

I was a person who loved Ferdium, but I think mow I will be removing it from all of my machines and using other means.

Such terrible PR in your response to Horia-M's message

**Reply by vraravam** [reply]
> For you to virtually demand that users move away from Windows 7 shoudl be illegal. It is certainly unethical for you to make these demands.

@steebs - please do keep in mind that @SpecialAro 's comment was never "demanding" or "illegal". This is a volunteer-run OSS project. We are all unpaid volunteers who contribute due to a sense of giving back to the community. So, in retrospect, I would consider your statement/demand as being very entitled.

We cannot support older versions of OSes, and if your stance is that you won't upgrade (which is very much in your realm/remit), feel free to find alternatives to Ferdium. Thanks for your usage of the tool till date.
